ui/perfherder/helpers.js

```diff
@@ -807,3 +812,6 @@ export const nudgeAlert = (dataPoint, towardsDataPoint) => {
   const alertId = dataPoint.alert.id;
   return update(getApiUrl(`/performance/alert/${alertId}/`), towardsDataPoint);
 };
+
+export const getRevisionUrl = (revision, projectName) =>
+  revision ? getJobsUrl({ repo: projectName, revision }) : '';
```

Comment:
It might be better to return plain text rather than an empty anchor tag, since if it's still clickable users might think it's broken (you could probably do this in render instead unless you know this will be used elsewhere).